examples/secret_manager/versions.tf

```diff
@@ -16,4 +16,14 @@
 
 terraform {
   required_version = ">= 0.12"
```

Comment:
Since we are updating these, this should actually be >=0.13 as thats what the modules support

Reply:
Changed in all versions.tf